spec/factories/mentor_profiles.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :mentor_profile do
+    user
+  end
+end
```

Comment:
Can we also add a trait to the user factory for the case of them being a mentor?